### PR TITLE
The big refractor utils

### PR DESCRIPTION
**Pull request by Adhikram**

### CODE REVIEW

File: services/content.py [hunk 1]
```diff
@@ -8,11 +8,10 @@
...
 from logger import errorLogger, infoLogger
 from services.constants import STATUS_DESC_MAP
-from myutils.send_request import send_request
-from myutils.dataNotFound import data_not_found
-from myutils.inputDm import take_input_dm
-# from utils import data_not_found, take_input_dm
-# from utils import data_not_found, send_request, take_input_dm
+from utils.send_request import send_request
+from utils.dataNotFound import data_not_found
+from utils.inputDm import take_input_dm
```
Comment (patheticGeek) [inline services/content.py:13]: dont use camel case, use snake case :snake:

File: services/group.py [hunk 2]
```diff
@@ -9,11 +9,9 @@
...
 from client import client
 from logger import errorLogger, infoLogger
-from myutils.time import get_seconds_till_weekday
-from myutils.send_request import send_request
-from myutils.dataNotFound import data_not_found
-# from utils import data_not_found
-# from utils import data_not_found, get_seconds_till_weekday, send_request
+from utils.time import get_seconds_till_weekday
+from utils.send_request import send_request
+from utils.dataNotFound import data_not_found
```
Comment (patheticGeek) [inline services/group.py:14]: Use snake case here too

File: utils/inputDm.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    try:
+
+        def check(message):
+            return message.channel == user.dm_channel and message.author == user
+
+        user_input = await client.wait_for("message", check=check, timeout=60)
+
+    except asyncio.TimeoutError:
+        await user.send("Sorry, You didn't reply on time.")
+        user_input = False
```
Comment (patheticGeek) [inline utils/inputDm.py:18]: Remove extra empty lines and return inside try catch instead of at end and use `None` instead of `False`

**Review by patheticGeek — CHANGES_REQUESTED** [review]
Noice work :+1:

File: api/request.py [hunk 4]
```diff
@@ -20,16 +18,16 @@ def stringifyQueryParams(params={}):
     for paramKey in params.keys():
         queryParams += f"{paramKey}={params[paramKey]}"
         if paramAt < paramsLength - 1:
-            queryParams += '&'
+            queryParams += "&"
         paramAt += 1
 
     return queryParams
```
Comment (kaydee0502) [inline api/request.py:25]: Too many camel cases :eyes:

Comment (patheticGeek) [reply api/request.py:25]: yeah ik, will be refactored further

**Pull request merged**